packages/orchestrator-ui-components/src/components/WfoSubscription/subscriptionDetailTabs.tsx

```diff
@@ -1,29 +1,48 @@
 import React from 'react';
 
-import { EuiIcon } from '@elastic/eui';
+import { WfoFilterTab } from '@/components';
+import { WfoCogFill } from '@/icons/WfoCogFill';
+import { WfoCubeSolid } from '@/icons/WfoCubeSolid';
+import { WfoPlayCircle } from '@/icons/WfoPlayCircle';
+import { WfoShare } from '@/icons/WfoShare';
 
-import { WfoFilterTab } from '../WfoFilterTabs';
 import { SubscriptionDetailTab } from './utils';
 
 export const subscriptionDetailTabs: WfoFilterTab<SubscriptionDetailTab>[] = [
-    {
-        id: SubscriptionDetailTab.SERVICE_CONFIGURATION_TAB,
-        translationKey: 'serviceConfiguration',
-        prepend: <EuiIcon type="submodule" />,
-    },
     {
         id: SubscriptionDetailTab.GENERAL_TAB,
         translationKey: 'general',
-        prepend: <EuiIcon type="devToolsApp" />,
+        prepend: (
+            <WfoCubeSolid
+                width="18"
+                height="18"
+                xPosition={-1}
+                yPosition={-1}
+            />
+        ),
+    },
+    {
+        id: SubscriptionDetailTab.SERVICE_CONFIGURATION_TAB,
+        translationKey: 'serviceConfiguration',
+        prepend: (
+            <WfoCogFill width="18" height="18" xPosition="-2" yPosition="-2" />
```

Comment:
I think it might be better to try to position the icon in the center instead of tweaking with magic numbers. When i remove the x and y position props the icon is positioned as: 
![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/b8f2216a-2cff-430d-a83c-a00dfda3493f)

I would suggest to try to move the icon in the center (vertically) in the span